src/components/20-molecules/dropdown/index.js

```diff
@@ -103,15 +103,16 @@ class AXADropdown extends NoShadowDOM {
 
   set value(newValue) {
     const {
-      state: { isControlled, value },
+      state: { isControlled, value, firstTime },
     } = this;
     // first value coming in indicates controlledness?
-    if (!isControlled && newValue !== undefined) {
+    if (!isControlled && newValue !== undefined && firstTime) {
```

Comment:
ShouldUpdate lifecycle is to late, for the adaptations?

Reply:
Unfortunately, a property-changing operation is needed *before* render() - currently implemented in shouldUpdate - hence moving the isControlled logic to after render() is too late.